storage/google/cloud/storage/blob.py

```diff
@@ -30,6 +30,9 @@
 import six
 from six.moves.urllib.parse import quote
 
+import google.auth.transport.requests as tr_requests
```

Comment:
Seems a little straining-at-gnats-swallowing-camels, given the imports just below.